pipprivates/install.py

```diff
@@ -34,11 +34,11 @@ def collect_requirements(fname, transform_with_token=None):
 
         # Rewrite private repositories that normally would use ssh (with keys in an agent), to using
         # an oauth key
-        elif tokens[0] == '-e' and tokens[1].startswith('git+git@github.com:'):
+        elif tokens[0] == '-e' and (tokens[1].startswith('git+git@github.com:') or tokens[1].startswith('git+https://github.com/')):
```

Comment:
`git+https` shouldn't be converted, because that's a public link. I think this `elif` should be split up:
```
elif tokens[0] == '-e':
    if tokens[1].startswith('git+git@github.com:'):
        vcs_url = 'git+https://{}:x-oauth-basic@github.com/{}'.format(
            transform_with_token, tokens[1][19:]) if transform_with_token else tokens[1]
    else:
        vcs_url = tokens[1]
    collected += [vcs_url]
```